steps/synopsys-vcs-sim/configure.yml

```diff
@@ -0,0 +1,40 @@
+#=========================================================================
+# Synopsys VCS -- Generic VCS Simulation
+#=========================================================================
+
+name: synopsys-vcs-sim
+
+#-------------------------------------------------------------------------
+# Inputs and Outputs
+#-------------------------------------------------------------------------
+
+inputs:
+  - adk
+  - design.vcs.v
+  - testbench.sv
+  - design.sdf
+  - design.args
+
+outputs:
+  - design.vpd
+
+#-------------------------------------------------------------------------
+# Commands
+#-------------------------------------------------------------------------
+
+commands:
+  - |
```

Comment:
Is the pipe needed?

Reply:
One of apparently 63 ways of getting multiline things in YAML.